_authors/chenxi-wang.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: <Chenxi Wang>
```

Comment:
Please remove the angled brackets <> around all of these entries.

Reply:
thanks and addressed